### ISSUE TITLE
James Project Feedback

### ISSUE THREAD

**Issue by BuiltByWalsh** [tyler-parker/police-brutality-project#1]
## Context
* I know this isn't the typical way to share feedback (usually done through a PR) but I figured this works for our use case.

## Kudos
* Putting your different react contexts in a shareable `/context` directory is sweet
* Love that you included a dark mode switcher! Woot woot!
* THE TIMELINE ANIMATIONS OH MY GAWWWW. Love it. ❤️. 

## Could be Better

- [ ] I would commit to either a `package-lock.json` or a `yarn.lock` but not both. Pick either NPM or yarn and charge forward owning that decision to pick one tool over the other!
- [ ] Sometimes you use `.js` file extensions and sometimes yo use `.jsx`. Consistency would be a nice to have!
- [ ] Architecture Thing: You have the `allStates.json` _(which I assume is a mock of what would normally be API data)_. And then the `States.js` which has an array of state data. I can't help but wonder if these data sets could be combined and _live in one place for ease of use_ and easier reasoning about the data set/source.

## Styling Ideas
Take these or leave these, this is just me jammin' on some stuff.
- [ ] Rather than using a `border-color` when you select a state, I would just nix that and use the CSS prop `cursor: pointer;` when the cursor moves over the text on the state. 
- [ ] It would be cool if the state background color went just a tad darker in dark mode. You can use the `useColorModeValue` hook from chakra to accomplish setting what the different colors should be for light vs dark.

**Comment by tyler-parker** [comment]
Totally agree on the feedback.

I'm not familiar with choosing one package over the other, but yeah I did notice that. I think it came from when I created the App from Chakra's template. I think they default to yarn. Or maybe I copied the wrong command 😅

I agree with the file extension consistency, I got a little lazy lol. But moving forward I'm gonna be better about that. 

Having those two state files was purely convenience because the `allStates.json`  came with the map library I installed and the `States.js` file was made because it had the abbreviation and the full state name, which is what the API uses to filter results. So I just copied that file from the innernet so I didn't have to populate the JSON with the full name of the state. 

I'll look into that border issue with the dropdown.

Yeahhhh, I ran into some issues using that hook within that MapChart component. React wasn't recognizing it for some reason. But maybe I just did it wrong. I'll have to revisit that.
